main.go

```diff
@@ -44,8 +44,8 @@ func main() {
 	flag.BoolVar(&keep, "keep", false, "Do not remove container image")
 	flag.BoolVar(&reboot, "reboot", false, "reboot if changed")
 	flag.Usage = func() {
-		fmt.Printf("pivot: version %s (%s)\n", version, commitHash)
-		fmt.Printf("Usage:\n")
+		fmt.Println("pivot: version %s (%s)\n", version, commitHash)
```

Comment:
Why switch to ``fmt.Println()`` *with* newlines?

Reply:
Yep, good catch, updated  :arrow_down: